### DISCUSSION TITLE
Unable to update min and max replicas

### DISCUSSION THREAD

**Discussion by faizanbashir** [kedacore/keda#4445] [Q&A / Need Help] [answered]
I cannot update the min and max replicas in an HPA ScaledObject. It keeps falling back to the initial value.

**Comment by JorTurFer** [comment, marked as answer]
> I cannot update the min and max replicas in an HPA ScaledObject. It keeps falling back to the initial value.

Hi,
Are you trying to update the ScaledObject or the HPA directly? If you are updating the HPA directly, you shouldn't do it as the HPA is something internal for KEDA and KEDA manages it (removing your manually applied changes), you should always modify the ScaledObject and never modify the HPA manually

**Reply by faizanbashir** [reply]
Thanks
